Dockerinstall

joelfriesen opened the pull request:
Init

Commit:
The basic theme and docker install - no plugins

joelfriesen committed:
Update README.md

changed readme to deal with a clean install

Commit:
Include needed plugins to get your site running

Commit:
Updates to plugins

Commit:
updates to ignore and readme

Commit:
Merge remote-tracking branch 'origin/dockerinstall' into dockerinstall

Commit:
CIOOS colours and site design

Commit:
Generic WordPress files that came with the image

Commit:
Changes and updates to the files.

 honing in the changes that experon didn't bring, and elementor overwrites.

Commit:
fix env

joelfriesen committed:
Delete .env

DEL ENV

Commit:
Working in the side bar

Commit:
Minor changes to the wp-config files

Commit:
form update

Commit:
Docker install cleanup

Commit:
Cleaned up the file structure of the theme package

Commit:
gitignore

Commit:
ignore kismet

Commit:
added plugins

Commit:
Theme

Commit:
removed submodule

Commit:
added theme

Commit:
added blank DB folder

Commit:
added it to git ignore

Commit:
Updated Readme

Commit:
CONFIG EXTRA

Commit:
Works with latest wordpress.

fostermh commented:
Amazing.

@steviewanders or @joelfriesen any suggestions on the best way to spin this PR up for testing? Or perhaps it is already on dev?

joelfriesen committed:
Update README.md

Updated instructions

joelfriesen committed:
Update README.md

Commit:
Turned off theme support for logo, added French / English logo in Customize
Cleaned up language options in menu
Added right fonts, cleaned up widget
cleaned up logo assets
added de-elementor

odumidika committed:
Updating footer styling, changed img files for atlantic

odumidika committed:
Update to main navigation styling

Commit:
My Local Commits

odumidika committed:
General updates and styling for Guttenburg home page

Commit:
Changes to navigation and H1 Heights

Commit:
Merge remote-tracking branch 'origin/odum' into Huey_Luis_and_the_New_CIOOS_branch

Commit:
Nav tweaks

Commit:
Pacific colours

Commit:
New functions

ADDED footer svg
MOVED colors into a section of their own to extract them easily for RA's
REMOVED a couple of unused var in SCSS, and grouped like colours

Commit:
More functions

ADDED footer svg
MOVED colors into a section of their own to extract them easily for RA's
REMOVED a couple of unused var in SCSS, and grouped like colours

Commit:
footer updates

Commit:
Merge branch 'dockerinstall' into Huey_Luis_and_the_New_CIOOS_branch

Commit:
Cudstom colours extracted

Added all RA colours to the CSS file at the bottom so I can use it in the customizer options
